### ISSUE TITLE
Various widgets don't seem to work in v2.2.1

### ISSUE THREAD

**Issue by Scherub** [mycumycu/X4-External-App#5]
Hi,

your mod is amazing! I've had v1.3.1 running properly in March. Then from April up until an hour ago I took a break from the game. When I started everything as usual, the game and your mod still worked. Then I saw the update message, so I just updated to v.2.2.1.

Unfortunately, there are a few things that don't work anymore. Starting with the missing 'Mission Offers' widget. I went on to delete the local storage, which at least solved this problem. Apparently the only widgets working are the 'Mission Offers' and the 'Logbook'.

The widgets 'Active mission' and 'Current research' just say 'none', while the 'Player profile' widget stays empty.

I looked at the python server, whether there is a new version, but nope. Am I missing something? Just for testing purposes, I started the old version 1.3.1 again and this version still displays the expected content within these widgets.

It would be a shame not to be able to use the new version, as I like the new widgets, particularly the 'goals' widget.

Do you have any idea how to solve it?

**Comment by Scherub** [comment]
Well, of course the error was on my side. :)

I've cloned your repo, added a few debug strings and built the project. After running it, I realized that various bits of the gameData object didn't match the data assigned to the widgets: 'playerProfile' instead of 'player', etc.

That reminded me that there was a second part of your mod that had to be updated. Ooops!

Anyway, it all works now. Thanks again for your great mod!
